Doppler.HtmlEditorApi/Controllers/CampaignsController.cs

```diff
@@ -94,6 +94,15 @@ public async Task<IActionResult> SaveCampaign(string accountName, int campaignId
                     Detail = $@"The content cannot be edited because status campaign is {campaignState.CampaignStatus}"
                 });
             }
+
+            if (campaignState.TestABCondition != TestABCondition.NoTestAB)
+            {
+                return new BadRequestObjectResult(new ProblemDetails()
+                {
+                    Title = "The campaign is AB Test",
+                    Detail = $@"The type of campaign with id {campaignId} is AB Test"
+                });
+            }
```

Comment:
It would be nice having a test for this new behavior.

Reply:
Done!